### PR TITLE
Add ballerina file-api example

### PR DESCRIPTION
**Pull request by Arshardh**

### CODE REVIEW

File: samples/ballerina-by-example/examples/file-api/file-api.bal [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    string s = blobs:toString(content, "utf-8");
+    //Print read content
+    system:println("file content: " + s);
+    files:close(target);
+
+    files:File source = {path : "/tmp/result.txt"};
+    files:File destination = {path : "/tmp/copy.txt"};
+    //Copy source file to destination
+    files:copy(source, destination);
+    system:println("file copied: /tmp/result.txt to /tmp/copy.txt");
```
Comment (chanakaudaya) [inline samples/ballerina-by-example/examples/file-api/file-api.bal]: This line goes out of the source code border within generated HTML.

File: samples/ballerina-by-example/examples/file-api/file-api.bal [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+    //Delete a file by giving the file struct.
+    files:delete(destination);
+    system:println("file deleted: /tmp/copy.txt");
+
+    destination = {path : "/tmp/move.txt"};
+    //Move source file to destination
+    files:move(source, destination);
+    system:println("file moved: /tmp/result.txt to /tmp/move.txt");
```
Comment (chanakaudaya) [inline samples/ballerina-by-example/examples/file-api/file-api.bal]: This line goes out of the source code border within generated HTML.

File: samples/ballerina-by-example/examples/file-api/file-api.bal (hunk 1 above)
Comment (Arshardh) [inline samples/ballerina-by-example/examples/file-api/file-api.bal]: fixed this

File: samples/ballerina-by-example/examples/file-api/file-api.bal (hunk 2 above)
Comment (Arshardh) [inline samples/ballerina-by-example/examples/file-api/file-api.bal]: fixed this

**Pull request merged**